packages/forms-web-app/src/services/pdf.service.js

```diff
@@ -97,7 +98,38 @@ const storePdfAppeal = async (appeal, fileName) => {
 	}
 };
 
+const storeTextAsDocument = async (submission, plainText, docType) => {
+	const log = logger.child({ uuid: uuid.v4() });
+
+	log.info('Storing PDF appeal document');
```

Comment:
'Attempting to store PDF document' - it might not be an appeal? (i.e. might be final comment)

Reply:
for clarity, I mean changing the log message to 'Attempting to store PDF document' (without the word appeal)